### PR TITLE
Portfolio desktop version 

### PR DESCRIPTION
**Pull request by mwenyoa**
**The following features were implemented in desktop version:**
- Add  Navigation for the desktop version
-  Add  media query  that changes the background image in desktop screen size
- Add Work section images , title, description and see project button
-  Add media query to about section toggle the content layout in desktop version
-  Add contact section  title, description and contact form and send message button

### CODE REVIEW

File: desktop-images/contactbg.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="1440" height="846" viewBox="0 0 1440 846" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0 846H1440V0H146C65.3664 0 0 65.3664 0 146V846Z" fill="#6070FF"/>
+</svg>
```
Comment (OLIPLICHE) [inline desktop-images/contactbg.svg:3]: - [ ] Please there is a break in your cards. check and fix them. (I checked them from 2 different screens)

p1

p4

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @mwenyoa, 

I love your Job! 👏 👏 👏 
There are some issues that you still need to work on to get your project approved. Well done my Sir 👍 .

## Highlights
- Linter result is green ✔️
- Commit messages are meaningful✔️
- The pull request have a descriptive title telling you what changes are inside ✔️
- The pull request have a short summary describing introduced changes in more detail ✔️
- node_modules folder is in the .gitignore ✔️
- Descriptive ReadMe

## Required Changes ♻️
- [ ] Make sure the `See Project` button takes you to the GitHub repository.

p2
- [ ] CHECK THE COMMENTS UNDER THE REVIEW

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mwenyoa** [comment]
@OLIPLICHE 
**Below is the implementation of requested changes:**

im1
im2

 - And the highlighted blue button is working as expected.

**Review by bimbolabuari — APPROVED** [review]
Hi @mwenyoa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**